js/main.js

```diff
@@ -12,30 +12,47 @@ const $form = document.querySelector('#form-id');
 $form.addEventListener('submit', subButton);
 
 function subButton(event) {
+  const $eform = document.querySelector('.eform');
   event.preventDefault();
-  const entry = {
-    entryId: data.nextEntryId,
-    title: event.target.elements.title.value,
-    photo: event.target.elements.myUrl.value,
-    notes: event.target.elements.notes.value
-  };
-  const $ul = document.querySelector('ul');
-  $ul.appendChild(renderEntry(entry));
+  let entry = {};
+
+  if (data.editing !== null) {
+
+    for (let i = 0; i < data.entries.length; i++) {
+      // eslint-disable-next-line eqeqeq
+      if (data.entries[i].entryId == data.editing) {
+        data.entries[i].title = $eform.children[0][0].value;
+        data.entries[i].photo = $eform.children[0][1].value;
+        data.entries[i].notes = $eform.children[0][2].value;
+      }
+    }
+    renderEntry(entry);
```

Comment:
This should be getting appended to the DOM. You'll need to replace the entry that currently exists on the DOM with this newly created entry